neurom/view/common.py

```diff
@@ -50,542 +39,311 @@
-            Default value is None.
-        yticks (Optional[list of ticks]): \
-            Defines the values of y ticks in the figure. \
-            If None the yticks will not be modified. \
-            For no_yticks set to []. \
-            Default value is None.
-        zticks (Optional[list of ticks]): \
-            Defines the values of z ticks in the figure. \
-            If None the zticks will not be modified. \
-            For no_zticks set to []. \
-            Default value is None.
-        tick_fontsize (Optional[int]): \
-            Defines the size of the ticks' font. \
-            Default value is 12.
-        xticks_arg (Optional[dict]): \
-            Defines the arguments that will be passsed \
-            into matplotlib as xticks arguments. \
-            Default value is None.
-        yticks_arg (Optional[dict]): \
-            Defines the arguments that will be passsed \
-            into matplotlib as yticks arguments. \
-            Default value is None.
-        zticks_arg (Optional[dict]): \
-            Defines the arguments that will be passsed \
-            into matplotlib as zticks arguments. \
-            Default value is None.
+        tick_fontsize (int): Defines the size of the ticks' font
+        xticks([list of ticks]): Defines the values of x ticks in the figure
+        xticks_arg(dict):  Passsed into matplotlib as xticks arguments
+        yticks([list of ticks]): Defines the values of y ticks in the figure
+        yticks_arg(dict):  Passsed into matplotlib as yticks arguments
+        zticks([list of ticks]): Defines the values of z ticks in the figure
+        zticks_arg(dict):  Passsed into matplotlib as zticks arguments
     """
-
-    # Definition of tick options
-    xticks = kwargs.get('xticks', None)
-    yticks = kwargs.get('yticks', None)
-    zticks = kwargs.get('zticks', None)
-
-    tick_fontsize = kwargs.get('tickfontsize', 12)
-
-    xticks_arg = kwargs.get('xticksarg', {})
-    yticks_arg = kwargs.get('yticksarg', {})
-    zticks_arg = kwargs.get('zticksarg', {})
-
     if xticks is not None:
         ax.set_xticks(xticks)
-        ax.xaxis.set_tick_params(labelsize=tick_fontsize, **xticks_arg)
+        xticks_args = dict_if_none(xticks_args)
+        ax.xaxis.set_tick_params(labelsize=tick_fontsize, **xticks_args)
 
     if yticks is not None:
         ax.set_yticks(yticks)
-        ax.yaxis.set_tick_params(labelsize=tick_fontsize, **yticks_arg)
+        yticks_args = dict_if_none(yticks_args)
+        ax.yaxis.set_tick_params(labelsize=tick_fontsize, **yticks_args)
 
     if zticks is not None:
         ax.set_zticks(zticks)
-        ax.zaxis.set_tick_params(labelsize=tick_fontsize, **zticks_arg)
+        zticks_args = dict_if_none(zticks_args)
+        ax.zaxis.set_tick_params(labelsize=tick_fontsize, **zticks_args)
 
 
-def plot_limits(ax, **kwargs):
+def plot_limits(ax, white_space):
     """Sets the limit options of a matplotlib plot.
```

Comment:
Yeah, it's huge, and I apologize for that.

I should rename this function to more clearly state what it does, which I will do.  If we want another function to directly set the limits, I can add that, it just probably wouldn't be called by any of the internal functions, so I don't know if it's something people would use